mmdet/models/dense_heads/yolo_head.py

```diff
@@ -303,7 +433,9 @@ def loss(self,
              gt_bboxes,
              gt_labels,
              img_metas,
-             gt_bboxes_ignore=None):
+             gt_bboxes_ignore=None,
+             target_maps_list=None,
```

Comment:
Under what cases will this parameter be passed? I see if static is run `get_bboxes_statically` branch.

Reply:
target_ maps_list and neg_ maps_list are the results of anchor matching, which is inefficient on IPU and will increase the compilation time by 2 hours.
So I put this part of the calculation in the dataloder. They do not need to rely on any middle outputs of yolo.
Just add gettargetsoutsideforyolo to the pipeline configuration, and the dataloder will output the target_ maps_list and neg_ maps_ list，When the yolo head receives these two data, it will automatically skip the calculation of anchor matching.
This configuration would also accelerate training on GPU.

Reply:
OK